### ISSUE TITLE
DOC: Build process followup

### ISSUE THREAD

**Issue by TomAugspurger** [statsmodels/statsmodels#2822]
Followup to https://github.com/statsmodels/statsmodels/pull/2816

- [ ] include `@savefig` decorators on all the example plots
- [ ] figure out why `duration.rst` breaks the execution on `index.rst` (and possibly elsewhere)
- [ ] integrate into Travis to auto-biuld
- [ ] cache the notebook execution step
- [ ] integrate statsmodels styling / template into HTML notebooks
- [ ] verify the output, make sure all the examples are executed and work
- [ ] fixup any compatibility things (including `make.bat`!)
- [ ] Pull in wiki examples?
- [ ] partial builds similar to pandas `make.py --single`

**Comment by josef-pkt** [comment]
about `duration.rst`: It was building for me last week, but AFAICS, there is at least one function that has the wrong module path. (`failed to import ...`)
But we have similar problems also for other modules.

**Comment by bashtage** [comment]
@TomAugspurger Are any of these important?  Some seem to have been done (duration, verify output)

**Comment by TomAugspurger** [comment]
No, I don't thing anything here is critical. Of these, I think building docs on Travis seems the most important, and IIRC you did some work on that?

**Comment by bashtage** [comment]
Yes, docs are built now.  They don't go anywhere, but it is easy to spot errors now (there is only 1 in master, the distributed notebook has an error).  Ideally it would be nice to push the dev docs automatically to somewhere useful. 

Don't know if `@savefig` is needed anywhere.  Notebook caching could also be useful for local work on docs (it is skipped completely on Windows for now)
